### PR TITLE
gpxsee: Add version 10.0

### PR DESCRIPTION
**Pull request by 2ion**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

As for config persistence, this project is using Qt5 and QSettings to store and manage user preferences. QSettings uses the Windows registry on Windows platforms which I am not able to capture in the manifest. The only other sort of persistence is the tile cache in AppData/Local (also not captured).

### CODE REVIEW

File: bucket/gpxsee.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "10.0",
+    "description": "GPS log file viewer and analyzer with support for GPX, TCX, KML, FIT, IGC, NMEA, SLF, SML, LOC, GPI, GeoJSON and OziExplorer files",
+    "homepage": "https://www.gpxsee.org/",
+    "license": "GPL3",
```
Comment (rashil2000) [inline bucket/gpxsee.json:5]:
```suggestion
    "license": "GPL-3.0-only",
```

File: bucket/gpxsee.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://jztkft.dl.sourceforge.net/project/gpxsee/Windows/GPXSee-10.0_x64.exe#/dl.7z",
```
Comment (rashil2000) [inline bucket/gpxsee.json:8]:
```suggestion
            "url": "https://downloads.sourceforge.net/project/gpxsee/Windows/GPXSee-10.0_x64.exe#/dl.7z",
```

File: bucket/gpxsee.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/gpxsee/files/Windows/",
+        "regex": "GPXSee-([0-9.]+)_x64\\.exe"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://jztkft.dl.sourceforge.net/project/gpxsee/Windows/GPXSee-$version_x64.exe#/dl.7z"
```
Comment (rashil2000) [inline bucket/gpxsee.json:26]:
```suggestion
                "url": "https://downloads.sourceforge.net/project/gpxsee/Windows/GPXSee-$version_x64.exe#/dl.7z"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
Any commandline arguments?

**Comment by 2ion** [comment]
> Any commandline arguments?

Yes and no.

 * Default set of Qt common arguments supported, none relevant as far as I can tell though: https://doc.qt.io/qt-5/qguiapplication.html#supported-command-line-options. QSettings behaviour not modifyable as a constructor defaulting to the native settings format (on Windows: Registry https://doc.qt.io/qt-5/qsettings.html#platform-specific-notes) is used: https://github.com/tumic0/GPXSee/blob/master/src/GUI/gui.cpp#L2156
 * Simple positional arguments specifying files to load into the app by path supported in application code also not relevant: https://github.com/tumic0/GPXSee/blob/master/src/GUI/app.cpp#L78

**Comment by rashil2000** [comment]
Okay, then we don't need bin field

**Pull request merged**